src/Shellbar/Shellbar.js

```diff
@@ -498,6 +503,4 @@ Shellbar.propDescriptions = {
     subtitle: 'Displays an application context. Should be used rarely.'
 };
 
-export { Shellbar as __Shellbar };
-
-export default withStyles(Shellbar, { cssFile: ['shellbar', 'product-switch'], fonts: true });
```

Comment:
should this file also `require('fundamental-styles/dist/fonts.css');` since fonts was true?

Reply:
Nope - there's a bunch of other components imported into this component that already require fonts. That was an oversight to include them in the first place with `withStyles`